experimental/packages/opentelemetry-instrumentation-grpc/src/utils.ts

```diff
@@ -110,3 +110,29 @@ export const _extractMethodAndService = (name: string): { service: string, metho
     method
   });
 };
+
+
+export function metadataCapture(type: 'request' | 'response', metadata: string[]) {
+  const normalizedMetadataAttributes = new Map(metadata.map(value => [value.toLowerCase(), value.toLowerCase().replace(/-/g, '_')]));
+
+  return (span: Span, getMetadata: (key: string) => undefined | string | string[] | Buffer) => {
```

Comment:
I didn't find the necessity to define a getter for the grpc metadata field.

```suggestion
  return (span: Span, metadata: grpc.Metadata) => {
```

This function can get the metadata instance and retrieve the metadata pairs with `metadata.get(key)`.

Reply:
True, made things a lot more readable without the getter, Fixed.